19/02/2018 03:08:46 PM 	Traceback (most recent call last):
			  File "main_modulo.py", line 29, in main
			    templateObj =  editoreplate(str(args['namedoc']))
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 8, in __init__
			    self.loadJson(file)
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 16, in loadJson
			    self.file =  json.load(open(path,'r+'))
			UnicodeEncodeError: 'ascii' codec can't encode character '\xf1' in position 45: ordinal not in range(128)
			
19/02/2018 04:40:56 PM 	Traceback (most recent call last):
			  File "main_modulo.py", line 29, in main
			    templateObj =  editoreplate(str(args['namedoc']))
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 8, in __init__
			    self.loadJson(file)
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 16, in loadJson
			    self.file =  json.load(open(path,'r+'))
			UnicodeEncodeError: 'ascii' codec can't encode character '\xf1' in position 43: ordinal not in range(128)
			
19/02/2018 04:41:05 PM 	Traceback (most recent call last):
			  File "main_modulo.py", line 29, in main
			    templateObj =  editoreplate(str(args['namedoc']))
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 8, in __init__
			    self.loadJson(file)
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 16, in loadJson
			    self.file =  json.load(open(path,'r+'))
			UnicodeEncodeError: 'ascii' codec can't encode character '\xf1' in position 43: ordinal not in range(128)
			
19/02/2018 04:41:14 PM 	Traceback (most recent call last):
			  File "main_modulo.py", line 29, in main
			    templateObj =  editoreplate(str(args['namedoc']))
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 8, in __init__
			    self.loadJson(file)
			  File "/var/www/html/demo/ocr_copia/edit_script.py", line 16, in loadJson
			    self.file =  json.load(open(path,'r+'))
			UnicodeEncodeError: 'ascii' codec can't encode character '\xf1' in position 43: ordinal not in range(128)
			
